news/2024/2024-03-03-4dmwc-2024-concludes.md

```diff
@@ -0,0 +1,171 @@
+
+> It is quite clear to me that the series has grown a lot compared to its first iterations. I think in terms of popularity it has always been a pretty popular one to watch due to the play field being a bit more even across all teams. But in terms of production quality and staff, it became huge compared to the first iteration.
+>
+> I think what's best about this though is that regardless of the growth it always felt like there was a pretty positive aura and environment regardless of how much more "preparations" were there.
+
+*What was the biggest challenge you had to face as a coordinator for this year’s 4DM?*
+
+> If you had asked me some weeks ago I would have probably said map selecting. That is always a pain regardless of the way you do it. Thankfully Danny, Komirin, and Demi managed that extremely well in the backend.
+>
+> But given you are asking now it would have to be dealing with the rescheduling issues we had on the last couple of rounds (finals and grand finals). Having Lunar New Year made it harder than it usually is and with time zones being so different it was pretty difficult to deal with.
+>
+> That and I guess dealing with people's feedback for map pools in general. It was hard to get our point across with our new pooling structure. Having LNs in rice maps or rice in LN maps was something difficult to discuss with feedback-givers
+
+*You've been a part of mania tournaments for about 10 years. What did you learn over the years of all the experience you have cultivated that makes you such a beloved member of the community?*
+
+> That's a tricky question, it feels so long ago... I think trying to be reasonable and talking with the community are the things I learned the most. We are all used to having big essays of text in mania and having async communication but I prefer to be more direct and have debates trying to get my points across, I believe that usually leads to more fruitful discussion and all parties tend to be happier afterward so long as the talks are reasonable and respectful.
+>
+> So yeah, being respectful, knowing the community somewhat to know how to talk to people and just trying to be friendly and nice with everyone.
+
+*How long do you see yourself wanting to continue being a huge part of osu!mania community until?*
+
+> I think "huge" is an overstatement nowadays in regard to how involved I am hah. Had I not been pulled by Danny and Komi in COE I don't know if 4DM would have even happened so, for that to be the case, my involvement by then couldn't be close to huge.
+>
+> I still enjoy commentating every once in a while so for now I don't think I will stop doing that for some more time. Hosting wise? I hope I don't do much more because I am not as reliable anymore, life is getting in the way (thankfully in a good way!) and it may be time to move on soon. I barely play anymore and just occasionally map due to it being fun to do, so as long as it stays fun I guess I will continue to do so.
+
```

Comment:
anymore or any more?

Reply:
anymore
any more spaces and it'd mean something different